src/function/table/system/pragma_table_info.cpp

```diff
@@ -50,6 +50,9 @@ static unique_ptr<FunctionData> PragmaTableInfoBind(ClientContext &context, Tabl
 	names.emplace_back("pk");
 	return_types.emplace_back(LogicalType::BOOLEAN);
 
+	names.emplace_back("unique");
+	return_types.emplace_back(LogicalType::BOOLEAN);
```

Comment:
`PRAGMA table_info` is specifically modelled after SQLite's equivalent `PRAGMA table_info` - so we don't want to add new columns here.